### PR TITLE
Changed the structure of the sentence.

### PR DESCRIPTION
**Pull request by Hamza-Imran75**
I'm not sure if it's only me (English is not my first language) but this sentence messed up my mind. I understood from the sentence that this requirement may disappear one day. To make it easier for other non-English speakers I changed the sentence structure.

Before: And it is not impossible that this requirement will one day disappear.

After: And this requirement may one day disappear.

Or we could do something like this: And it is possible that this requirement will one day disappear.

### CODE REVIEW

**Comment by rrousselGit** [comment]
I wanted to show that it's not certain that such a change will happen.  
I'm fine with simplifying the sentence. But then I'd probably use `might` instead of `may`.

File: website/docs/riverpod_for_provider_users.mdx [hunk 1]
```diff
@@ -63,8 +63,7 @@ to work:
 - Instead of extending `StatefulWidget`, with Riverpod you should extend
   `ConsumerStatefulWidget`.
 
-But this inconvenience is fairly minor in the grand scheme of things. And it
-is not impossible that this requirement will one day disappear.
+But this inconvenience is fairly minor in the grand scheme of things. And this requirement may one day disappear.
```
Comment (rrousselGit) [inline website/docs/riverpod_for_provider_users.mdx:66]:
```suggestion
But this inconvenience is fairly minor in the grand scheme of things. And this
requirement might one day disappear.
```

**Comment by rrousselGit** [comment]
Thanks for the feedback :)

File: website/docs/riverpod_for_provider_users.mdx (hunk 1 above)
Comment (Hamza-Imran75) [reply website/docs/riverpod_for_provider_users.mdx]: Oh makes sense.

**Pull request merged**